tests/sklearn/test_sklearn_autolog.py

```diff
@@ -1462,3 +1462,30 @@ def test_metric_computation_handles_absent_labels():
     assert not metrics
     assert tags == get_expected_class_tags(model)
     assert MODEL_DIR in artifacts
+
+
+def test_autolog_disabled_on_sklearn_cross_val_api():
+    mlflow.sklearn.autolog()
+    from sklearn import linear_model
+    from sklearn.model_selection import cross_validate, cross_val_predict, cross_val_score
+
+    def assert_autolog_disabled(run_):
+        params, metrics, tags, artifacts = get_run_data(run_.info.run_id)
+        assert params == {} and metrics == {} and tags == {} and artifacts == []
```

Comment:
```suggestion
        assert params == {}
        assert metrics == {}
        assert tags == {}
        assert artifacts == []
```

Nit: can we split this line to assert one item at a time to make it easier to debug errors?